### PR TITLE
Update VSCode debugging guide

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
$subject.

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md [hunk 1]
```diff
@@ -102,25 +102,30 @@ Follow the steps below to start a remote debug session.
...
 
-    - Debugging a Ballerina package/ single file: 
+    - Debugging a Ballerina package or a single file: 
 
     ```bash
     bal run --debug <DEBUGGEE_PORT> <BAL_FILE_PATH/PACKAGE_PATH>
     ```
 
-    - Debugging ballerina tests: 
+   - Debugging Ballerina executable jar:
```
Comment (praveennadarajah) [inline learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:111]:
```suggestion
   - Debugging Ballerina executable:  
```

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md [hunk 2]
```diff
@@ -102,25 +102,30 @@ Follow the steps below to start a remote debug session.
...
 
     ```bash
     bal run --debug <DEBUGGEE_PORT> <BAL_FILE_PATH/PACKAGE_PATH>
     ```
 
-    - Debugging ballerina tests: 
+   - Debugging Ballerina executable jar:  
+
+    ```bash 
+    bal run --debug <DEBUGGEE_PORT> <JAR_FILE_PATH>
```
Comment (praveennadarajah) [inline learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:114]:
```suggestion
    bal run --debug <DEBUGGEE_PORT> <EXECUTABLE_PATH>
```

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md (hunk 1 above)
Comment (NipunaRanasinghe) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:111]: Executable jar looks better to me, as using just "executable" might leave some doubt (executable code, an executable file, or an executable program?)

Comment (NipunaRanasinghe) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:111]: @praneesha WDYT?

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md (hunk 2 above)
Comment (NipunaRanasinghe) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:114]: https://github.com/ballerina-platform/ballerina-dev-website/pull/2189#discussion_r636828843

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md (hunk 1 above)
Comment (praneesha) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:111]: +1 say "executable JAR". Note the uppercase letters in the word "JAR".

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md (hunk 2 above)
Comment (praneesha) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:114]: Shall we say `<EXECUTABLE_JAR_FILE_PATH>`?

Comment (NipunaRanasinghe) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:114]: @praneesha +1

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md (hunk 1 above)
Comment (NipunaRanasinghe) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md]: Updated in f8d2254

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md (hunk 2 above)
Comment (NipunaRanasinghe) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md]: Updated in f8d2254

**Review by praneesha — APPROVED** [review]

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: ballerina-tooling-guide-left-nav-pages-swanlake
 title: Run and Debug
+description: Describes the existing Ballerina run and debugging support in vscode
```
Comment (praneesha) [inline learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md:4]:
```suggestion
description: Describes the existing run and debugging functionalities provided by Ballerina in Visual Studio Code. 
```

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/using-the-debugging-features.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: ballerina-tooling-guide-left-nav-pages-swanlake
 title: Using the Debugging Features
+description: Describes the existing ballerina debugging features in vscode
```
Comment (praneesha) [inline learn/tooling-guide/visual-studio-code-extension/run-and-debug/using-the-debugging-features.md:4]:
```suggestion
description: Describes the existing Ballerina debugging features provided in Visual Studio Code.
```

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md (hunk 3 above)
Comment (NipunaRanasinghe) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md]: Updated in 7025c57

File: learn/tooling-guide/visual-studio-code-extension/run-and-debug/using-the-debugging-features.md (hunk 4 above)
Comment (NipunaRanasinghe) [reply learn/tooling-guide/visual-studio-code-extension/run-and-debug/using-the-debugging-features.md]: Updated in 7025c57

**Pull request merged**